modules/50_damages/KWLike/postsolve.gms

```diff
@@ -9,37 +9,91 @@
 * Damage function based on Kalkuhl & Wenz (2020)
 * time index mapping from supplement to code:  tall = t ; tall2 = t' ; tall3 = t''
 
+display pm_regionalTemperature;
```

Comment:
all these parts are frankly quite difficult to review. It would be appreciated if you added some comments on what they mean

Reply:
I took it all out here as this is not working anyway, and is replaced by the new realization KW_SE. There I put some comments.